edx_api/certificates/models_test.py

```diff
@@ -121,3 +122,15 @@ def test_download_url(self):
     def test_grade(self):
         """Test for grade"""
         assert self.certificate.grade == "0.98"
+
+    def test_is_passing(self):
+        """Test for is_passing"""
+        assert self.certificate.is_passing is True
+
+    def test_created(self):
+        """Test for created"""
+        assert self.certificate.created == parser.parse("2015-07-31T00:00:00Z")
```

Comment:
this should assert something like 
`assert self.certificate.created == datetime(<the time>)` so we are sure of what the result is.